website/docs/addon_3dequalizer_admin.md

```diff
@@ -12,3 +12,17 @@ import versions from '@site/docs/assets/json/Ayon_addons_version.json'
 <ReactMarkdown>
 {versions.Equalizer_Badge}
 </ReactMarkdown>
+
+## Overview
+
+To get 3DEqualizer (3DE) addon running, we need to instal Qt Python bindings - for that we are using pre-launch hook that is automatically installing PySide2/6 into (3DE) Python. You can use ```TDE4_HOME``` environment variable to set the location, or it will be detected from the AYON settings.
+
+:::warning Python Bindings
+This automatic **PySide2/6** installation might fail if 3DE is used from protected location (either network drive or some user write restricted location). If this happens, you need to install PySide there manually.
+:::
+
+### Environment
+
+3DE Addon is using ```AYON_TDE4_HEARTBEAT_INTERVAL``` environment variable to control how often it should yield control from Qt processes back to 3DE. This value can influence stability of 3DE and its responsivness. Default value is 500.
+
+AYON is also using ```PYTHON_CUSTOM_SCRIPTS_3DE4``` environment variable to install itself to 3DE menu. It is appending to it but if you overwrite this variable using some startup script or rez, AYON menu won't be shown in 3DE.
```

Comment:
```suggestion
AYON is also using ```PYTHON_CUSTOM_SCRIPTS_3DE4``` environment variable to install itself to 3DE menu. This variable appends to AYON but if you overwrite the variable using some startup script or rez, AYON menu won't be shown in 3DE.
```

Reply:
Thank you. it should be resolved in https://github.com/ynput/ayon-documentation/pull/245/commits/54391d84509d492944b555a2dd5e4501a1d60b72